receiver/googlecloudspannerreceiver/internal/metadata/metricsdatapoint.go

```diff
@@ -114,6 +116,44 @@ func (mdp *MetricsDataPoint) toDataForHashing() dataForHashing {
 	}
 }
 
+// Convert row_range_start_key label of top-lock-stats metric from format "sample(key1, key2)" to "sample(hash1, hash2)"
+func parseAndHashRowrangestartkey(key string) string {
+	startIndexKeys := strings.Index(key, "(")
+	substring := key[startIndexKeys+1 : len(key)-1]
+	hashedKey := key[:startIndexKeys+1]
```

Comment:
Done . Updated PR is at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16343 because this one got closed because of inactivity